src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java

```diff
@@ -82,14 +80,6 @@ public void saveWithActorAndCountry_ok() {
         Assert.assertTrue(actualOptional.isPresent());
         Movie actual = actualOptional.get();
         Assert.assertNotNull(actual);
-        Assert.assertEquals(1L, actual.getId().longValue());
-        Assert.assertEquals(shawshankRedemption.getTitle(), actual.getTitle());
-        Assert.assertNotNull(actual.getActors());
-        Assert.assertEquals(1, actual.getActors().size());
-        Assert.assertEquals(morganFreeman.getName(), actual.getActors().get(0).getName());
-        Assert.assertNotNull(actual.getActors().get(0).getCountry());
-        Assert.assertEquals(1, actual.getActors().get(0).getCountry().getId().longValue());
-        Assert.assertEquals(usa.getName(), actual.getActors().get(0).getCountry().getName());
```

Comment:
I cannot approve your solution because you have modified the test
let's please restore the original state of this test class